LondonBritishLibrary/orient/BLorient796.xml

```diff
@@ -32,9 +33,405 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                       <locus from="30v"></locus>
+                       <title type="complete" xml:lang="gez">ጸሎት፡ በእንተ፡ እለ፡ ይነግድ፡</title>
+                       <textLang mainLang="gez"></textLang>
+                     </msItem>
+                     <msItem xml:id="ms_i1.3.4">
+                       <locus from="31r"></locus>
+                       <title type="complete" xml:lang="gez">ጸሎት፡ በእንተ፡ ዝናማት፡</title>
+                       <textLang mainLang="gez"></textLang>
+                     </msItem>
+                     <msItem xml:id="ms_i1.3.5">
+                       <locus from="31r"></locus>
+                       <title type="complete" xml:lang="gez">ጸሎት፡ በእንተ፡ ፍሬ፡ ምድር፡</title>
+                       <textLang mainLang="gez"></textLang>
+                     </msItem>
+                     <msItem xml:id="ms_i1.3.6">
+                       <locus from="31v"></locus>
+                       <title type="complete" xml:lang="gez">ጸሎት፡ በእንተ፡ ማያተ፡ አፍላግ፡</title>
+                       <textLang mainLang="gez"></textLang>
+                     </msItem>
+                     <msItem xml:id="ms_i1.3.7">
+                       <locus from="31v"></locus>
+                       <title type="complete" xml:lang="gez">ጸሎት፡ በእንተ፡ ንጉሥ፡</title>
+                       <textLang mainLang="gez"></textLang>
+                       <note>Here the name of <persName ref="PRS5617IyasuII"></persName>
+                     is mentioned.</note>
+                     </msItem>
+                     <msItem xml:id="ms_i1.3.8">
+                       <locus from="31v"></locus>
+                       <title type="complete" xml:lang="gez">ጸሎት፡ በእንተ፡ እለ፡ ያበውኡ፡
+                       መባዐ፡</title>
+                       <textLang mainLang="gez"></textLang>
+                     </msItem>
+                     <msItem xml:id="ms_i1.3.9">
+                       <locus from="32r"></locus>
+                       <title type="complete" xml:lang="gez">ጸሎት፡ በእንተ፡ ንኡሰ፡ ክርስቲያን፡</title>
+                       <textLang mainLang="gez"></textLang>
+                     </msItem>
+                     <msItem xml:id="ms_i1.3.10">
+                       <locus from="32r"></locus>
+                       <title type="complete" xml:lang="gez">ጸሎት፡ በእንተ፡ ሰላም፡</title>
+                       <textLang mainLang="gez"></textLang>
+                     </msItem>
+                     <msItem xml:id="ms_i1.3.11">
+                       <locus from="32v"></locus>
+                       <title type="complete" xml:lang="gez">ጸሎት፡ በእንተ፡ ማኅበር፡</title>
+                       <textLang mainLang="gez"></textLang>
+                     </msItem>
+                     <msItem xml:id="ms_i1.3.12">
+                       <locus from="32v"></locus>
+                       <title type="complete" xml:lang="gez">ጸሎት፡ በእንተ፡ ጳጳስ፡</title>
+                       <textLang mainLang="gez"></textLang>
+                       <note>Here Mark (1737-61) is mentioned.</note>
```

Comment:
My wikipedia research left me clueless on who he would be , I found no corresponding pope or metropolitan with this name and dates. @DenisNosnitsin1970 can you help?

Reply:
It is not clear how Mark is referred to, as papas or liqa papas, as both can be mentioned in the supplication, if I remember well; but if there is no Ethiopian metropolitan and no Coptic Patriarch Mark fitting the years, more or less, I do not know.